WalletWasabi.Tests/UnitTests/ViewModels/UiContext/NullClipboard.cs

```diff
@@ -1,17 +1,15 @@
 using System.Threading.Tasks;
-using Avalonia.Input;
-using Avalonia.Input.Platform;
 
 namespace WalletWasabi.Fluent.Models.UI;
```

Comment:
For me, there is a warning for the namespace also.

I can't add a suggestion here, so code snippet:

```
using WalletWasabi.Fluent.Models.UI;

namespace WalletWasabi.Tests.UnitTests.ViewModels.UiContext;
```